pkg/util/util.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017 Kinvolk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import "crypto/sha256"
+
+func MakeSha256(data []byte) []byte {
```

Comment:
This is now only used for the tests. This should be removed. We can do tests without hashing.